spec/resque_mailer_spec.rb

```diff
@@ -122,6 +122,20 @@ class CannotConnectError < RuntimeError; end
     end
   end
 
+  describe '#deliver_now' do
+    it 'has method_alias to #deliver' do
+      @message_decoy = Rails3Mailer.test_mail(Rails3Mailer::MAIL_PARAMS)
+      @message_decoy.method(:deliver_now).receiver == @message_decoy.method(:deliver).receiver
```

Comment:
Enter the multiverse folks 
![mult](https://cloud.githubusercontent.com/assets/39729/20851696/df74f3da-b8af-11e6-85e2-1d22e30791a4.gif)